isisd/isis_nb_config.c

```diff
@@ -2851,6 +2838,7 @@ int lib_interface_isis_mpls_holddown_modify(struct nb_cb_modify_args *args)
 
 		if (circuit->ldp_sync_info == NULL)
 			isis_ldp_sync_if_init(circuit, isis);
+		assert(circuit->ldp_sync_info != NULL);
```

Comment:
It's a shame coverity is reporting this as an issue, since there is no path through which `circuit->ldp_sync_info` can be null after the init call...